native/src/test/scala/cats/native/tests/FutureSuite.scala

```diff
@@ -0,0 +1,65 @@
+package cats.native.tests
+
+import cats.kernel.laws.discipline.{MonoidTests => MonoidLawTests, SemigroupTests => SemigroupLawTests}
+import cats.kernel.{Eq, Semigroup}
+import cats.laws.discipline._
+import cats.laws.discipline.arbitrary._
+import cats.syntax.either._
+import cats.tests.{CatsSuite, ListWrapper}
+import org.scalacheck.Arbitrary.arbitrary
+import org.scalacheck.rng.Seed
+import org.scalacheck.{Arbitrary, Cogen}
+
+import scala.concurrent.{Await, ExecutionContextExecutor, Future}
+import scala.concurrent.duration._
+
+class FutureSuite extends CatsSuite {
+  val timeout = 3.seconds
+
+  // TODO: We shouldn't do this! See: https://github.com/scala-js/scala-js/issues/2102
```

Comment:
Is this still accurate for Scala Native?

Reply:
It seems so. Using global ExecutionContext resulted in intermittent test failures.

The actual test suite is copied from the JVM platform project, modified with with synchronous ExecutionContext replacing the global import.